atomic_reactor/manifest_util.py

```diff
@@ -0,0 +1,205 @@
+"""
+Copyright (c) 2019 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+
+from __future__ import unicode_literals, absolute_import
+import json
+import requests
+from copy import deepcopy
+
+from atomic_reactor.plugin import PluginFailedException
+from atomic_reactor.plugins.pre_reactor_config import get_registries
+from atomic_reactor.util import RegistrySession, ManifestDigest
+from atomic_reactor.constants import (MEDIA_TYPE_DOCKER_V2_SCHEMA2,
+                                      MEDIA_TYPE_DOCKER_V2_MANIFEST_LIST, MEDIA_TYPE_OCI_V1,
+                                      MEDIA_TYPE_OCI_V1_INDEX)
+
+
+class ManifestUtil(object):
+    manifest_media_types = [
```

Comment:
Better to make this class attribute a tuple so that it's immutable.

I realised this is pre-existing code, but since we're moving it around...